src/rard/research/forms.py

```diff
@@ -622,7 +624,9 @@ def __init__(self, *args, **kwargs):
         if original_text and original_text.pk:
             original_text.update_content_mentions()
             if original_text.apparatus_criticus_items.count() > 0:
```

Comment:
Is there a reason this needs to come before the call to super().__init__ ? Just that it probably makes sense to follow the same order as above where `ennableApparatusCriticus` is added to the class list after `enableCKEditor`. Also I don't know if += would work here if widget.attrs["class"] potentially doesn't exist as a key yet. Have you tested it?

Reply:
Ah yeah, the failing pytest check seems to confirm this.